package.json

```diff
@@ -3,17 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
```

Comment:
i see these in the lock file, are they overriden somehow?
`renderkid.ansi-regex 2.1.1`
`renderkid.strip-ansi.ansi-regex ^2.0.0`

Reply:
yup, you're right. attempting to override